### ISSUE TITLE
Is it possible to style the "New" button for array type?

### ISSUE THREAD

**Issue by cheukwong** [networknt/react-schema-form#84]
I have looked at Array.js. It looks like the style field is ignored in the simplearray.json example.

```
  "style": {
                "add": "btn-success"
            },
```

**Comment by stevehu** [comment]
@cheukwong Yes. You are right. The examples were built a long time ago and we are planning to upgrade to the latest version of react-schema-form. Let's keep this issue open so that we can address at the same time. Thanks for bringing it up.
